setup.py

```diff
@@ -5,15 +5,15 @@
 requirements = [
     'createrepo_c~=0.13',
     'productmd',
-    'pulpcore-plugin~=0.1rc5',
+    'pulpcore-plugin~=0.1rc6',
```

Comment:
I'd probably recommend pinning to 0.1.0rc6. I foresee more breaking changes coming in pulpcore between now and GA.

Reply:
Updated